packages/bigquery/package.json

```diff
@@ -31,7 +31,7 @@
   ],
   "main": "./src/index.js",
   "files": [
-    "./src/*",
+    "src",
```

Comment:
This is required, otherwise packages are published without any files from `src`... which is why the first usable version of each module starts at `0.1.1`.